UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1001,16 +1004,20 @@ public void CalculateOverallHealth()
 				currentHealth -= implant.TotalDamageWithoutOxyCloneRadStam;
 			}
 
-			if (brain == null || brain.RelatedPart.Health < -100 || brain.RelatedPart.TotalModified == 0)
-			{
-				currentHealth -= 200;
-				healthStateController.SetOverallHealth(currentHealth);
-				CheckHeartStatus();
-				return;
-			}
-			else
+			if (DoesNotRequireBrain == false)
 			{
-				currentHealth -= brain.RelatedPart.Oxy;
+				if (brain == null || brain.RelatedPart.Health < -100 || brain.RelatedPart.TotalModified == 0)
+				{
+					currentHealth -= 200;
+					healthStateController.SetOverallHealth(currentHealth);
+					CheckHeartStatus();
+					return;
+				}
+				else
+				{
+
+					currentHealth -= brain.RelatedPart.Oxy;
```

Comment:
why are we subtracting oxygen?

Reply:
It's indicates that your brain is taking oxygen damage